enforcer-parent/enforcer/src/test/java/org/wso2/choreo/connect/enforcer/backend/jwt/BackendJwksTest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.enforcer.backend.jwt;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.wso2.choreo.connect.discovery.config.enforcer.Config;
+import org.wso2.choreo.connect.discovery.config.enforcer.JWTGenerator;
+import org.wso2.choreo.connect.discovery.config.enforcer.Keypair;
+import org.wso2.choreo.connect.enforcer.config.ConfigHolder;
+
+import java.io.File;
+
+public class BackendJwksTest {
+
+    private final String resourcePath = BackendJwksTest.class.getProtectionDomain()
+            .getCodeSource().getLocation().getPath();
+    private final String keyStore = "keystore";
+    private final String keyStoreSingleCert = "singleCert";
+    private final String keyStoreMultiCert = "multipleCerts";
+    private final String cert = "cert.pem";
+    private final String additionalCert = "mg.pem";
+
+    @Test
+    public void DefaultConfig() {
+        String certPath = resourcePath + keyStore + File.separator + keyStoreSingleCert + File.separator + cert;
+        ConfigHolder configHolder = ConfigHolder.getInstance();
+        JWTGenerator jwtGenerator = JWTGenerator.newBuilder()
+                .setEnable(true)
+                .addKeypairs(Keypair.newBuilder()
+                        .setPublicCertificatePath(certPath)
+                        .setUseForSigning(true)
+                        .buildPartial())
+        .build();
```

Comment:
Seems JWTGenerator is common. For the test in line 58 also. Let's keep that as a class level variable.